content/articles/compiled-redesign.md

```diff
@@ -0,0 +1,141 @@
+the web accessibility standards, and to improve its overall user experience
+inclusively on all devices."
+shortlede: "This site’s latest iteration has improved design and WCAG 2.0
+compliance."
+poster: "poster-compiled-v3.jpg"
+socmediaimg: ""
+hiliteimg: "poster-compiled-v3.jpg"
+poster_sourceurl: ""
+poster_source: ""
+bookendanimal: "dragon"
+---
+
+In March, I restructured and redesigned the CTL DevTeam’s website _CompilED_ to
+meet the web accessibility standards, and to improve its overall user
+experience inclusively on all devices.
+
+This latest iteration of _CompilED_ follows the
+[Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/TR/WCAG20/)
+level AA, and it is much more technically and usably accessible than the
+previous versions. It is keyboard accessible with improved focus styling. The
+site is also seamlessly adaptive to multi-size screen devices—smartphones,
+tablets, and desktops. The comprehensive design has also improved the
+readability of the site’s content.
+
+In my work at the CTL, I consider accessibility as an integral part of my own
+development process. Lately, I’ve been mulling over an effective, broader
+integration of accessibility thinking, process, and policy across different
+groups in the whole department, as part of our commitment to accessibility,
+universal design, and inclusion in higher education.
+
+[_CompilED_ redesign in 2016](/articles/rebuilding-compiled/) afforded some
+accessibility since my grasp of WCAG standards was limited at the time. Three
+years later with more experience and grounded understanding, I redesigned
+_CompilED_ as an exercise to develop a website that conforms very well to the
+WCAG. This experience led me to best practices and a workable flow that I can
+share with my team for other development processes.
+
+## Steps on improving CompilED’s accessibility
+
+### User experience design
+
+I thought carefully about _CompilED_ content delivery and comparable user
+experience on different medium, from visual to auditory experience. Much of the
+UX design centered around the integrity of the visual UI and the browser’s
+rendering of the accessibility tree for screen readers. The elements and
+content on the pages were regrouped according to function, reordered according
+to priority, and restacked for responsive design. This regrouping and
+reordering was done so that the structure on each page make sense to all users
+regardless of their choice of devices—desktop, mobile, or assistive
+technologies.
+
+I’ve written on similar design process for the
+[redesign of PHTC image map](/articles/a11y-rwd-imagemap/) and
+[two Dental School projects](/articles/deconstructing-accessibility/).
+
+### Semantic HTML and ARIA
+
+In her demonstration on 
+[how a screen reader user accesses the web](https://www.smashingmagazine.com/2019/02/accessibility-webinar/),
+accessibility engineer [Léonie Watson](https://tink.uk/) recommended using
```

Comment:
Maybe change 'recommended' to 'recommends' to match the tense of 'clarify'; that and I'm sure she still continues to recommend such practices.

Reply:
"clarify" is the verb after "to, that is "to improve and to clarify". I added a "to" next to "clarify"